apps/ethvm/src/processor/index.ts

```diff
@@ -0,0 +1,5 @@
+import processTxs from '@app/processor/txProcessor'
```

Comment:
Remove this lines and use export * from '@app/processor/txProcessor' syntax

Reply:
done